pages/common/dart.md

```diff
@@ -0,0 +1,28 @@
+# dart
+
+> The tool for managing Dart projects.
+> More information: <https://dart.dev/tools/dart-tool>.
+
+- Create a new project:
```

Comment:
```suggestion
- Create a new project in the current directory:
```

Reply:
It doesn't. It creates a new directory containing the new project.

Reply:
where?

Reply:
`./{{project_name}}`

Reply:
Hmm, that's not so easy to say in one sentence. The only thing I can come up with:
> Create a new project direcory in the current directory:

Reply:
How about:
> Create a new project directory in current path

Reply:
Well, that seems kinda confusing to me, because it makes it sound like it *just* creates a directory.

Reply:
I vote to leave the description as-is.

Reply:
Sounds like the best option.

Reply:
I like @marchersimon's suggestion here of `Create a new project directory in the current directory:`.

Reply:
"Create a directory in the current directory for a new project"?

Just another idea.

Reply:
@pixelcmtd what do you think?
```suggestion
- Initialize a new Dart project in a directory of the same name:
```

Reply:
I think we could leave it as is, but I'm fine with this one also.

Reply:
> @pixelcmtd what do you think?

I personally really don't like unnecessarily long descriptions and this whole review seems to me like a competition on who can come up with the longest, worst-sounding one.